Q/Qt/build_tarballs.jl

```diff
@@ -0,0 +1,154 @@
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "qt5"
+version = v"5.14.1"
+
+# Collection of sources required to build qt5
+sources = [
+    ArchiveSource("https://download.qt.io/official_releases/qt/5.14/5.14.1/single/qt-everywhere-src-5.14.1.tar.xz",
+                  "6f17f488f512b39c2feb57d83a5e0a13dcef32999bea2e2a8f832f54a29badb8"),
+    ArchiveSource("https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.14.sdk.tar.xz",
+                  "0f03869f72df8705b832910517b47dd5b79eb4e160512602f593ed243b28715f")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir build
+cd build/
+
+commonoptions=" \
+-opensource -confirm-license \
+-skip qtactiveqt -skip qtandroidextras -skip qtcanvas3d -skip qtconnectivity -skip qtdatavis3d -skip qtdoc -skip qtgamepad \
+-skip qtnetworkauth -skip qtpurchasing -skip qtremoteobjects -skip qtscript -skip qtscxml -skip qtsensors -skip qtserialbus \
+-skip qtserialport -skip qtspeech -skip qtvirtualkeyboard -skip qtlocation -skip qtwayland -skip qtwebchannel -skip qtwebengine \
+-skip qtwebglplugin -skip qtwebsockets -skip qtwebview  -skip qttools -nomake examples -release \
+"
+
+case "$target" in
+
+	*86*linux*)
+        export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$prefix/share/pkgconfig
+        ../qt-everywhere-src-5.14.1/configure -L $prefix/lib -I $prefix/include \
+            -prefix $prefix $commonoptions \
+            -skip qtwinextras -system-xcb -fontconfig
+        ln -s /lib64/libc.so.6 /lib64/libc.so
+        ;;
+        
+	*apple-darwin*)
+        cd $WORKSPACE/srcdir/MacOSX10.14.sdk
+        rm -rf /opt/$target/$target/sys-root/System
+        rsync -a usr/* /opt/$target/$target/sys-root/usr/
+        cp -a System /opt/$target/$target/sys-root/
+        
+        cd $WORKSPACE/srcdir/qt-everywhere-src-5.14.1/qtbase
+        
+        cat <<EOT > mkspecs/features/mac/default_pre.prf
+CONFIG = asset_catalogs rez \$\$CONFIG
+load(default_pre)
+
+QMAKE_ASSET_CATALOGS_APP_ICON = AppIcon
+
+macx-xcode:qtConfig(static): \\
+    QMAKE_XCODE_DEBUG_INFORMATION_FORMAT = dwarf
+
+QMAKE_XCODE_LIBRARY_SUFFIX_SETTING = QT_LIBRARY_SUFFIX
+
+xcode_copy_phase_strip_setting.name = COPY_PHASE_STRIP
+xcode_copy_phase_strip_setting.value = NO
+QMAKE_MAC_XCODE_SETTINGS += xcode_copy_phase_strip_setting
+EOT
+        
+        sed -i '1s;^;QMAKE_MAC_SDK.macosx.Path = '"/opt/$target/$target/sys-root"'\
+        QMAKE_MAC_SDK.macosx.SDKVersion = '"10.14"'\
+        QMAKE_MAC_SDK.macosx.PlatformPath = '"/opt/$target"'\n;' 'mkspecs/features/mac/sdk.prf'
+        echo "" >  mkspecs/features/mac/no_warn_empty_obj_files.prf
+        
+        sed -i 's/-fuse-ld=x86_64-apple-darwin14/-fuse-ld=\/opt\/bin\/x86_64-apple-darwin14-ld/g' /opt/bin/x86_64-apple-darwin14-clang++
+        sed -i 's/-fuse-ld=x86_64-apple-darwin14/-fuse-ld=\/opt\/bin\/x86_64-apple-darwin14-ld/g' /opt/bin/x86_64-apple-darwin14-clang
```

Comment:
Did you know that you can more simply do
```suggestion
        sed -i 's?-fuse-ld=x86_64-apple-darwin14?-fuse-ld=/opt/bin/x86_64-apple-darwin14-ld?g' /opt/bin/x86_64-apple-darwin14-clang++
        sed -i 's?-fuse-ld=x86_64-apple-darwin14?-fuse-ld=/opt/bin/x86_64-apple-darwin14-ld?g' /opt/bin/x86_64-apple-darwin14-clang
```
?  The separator in the `sed` expression can actually be any character that doesn't appear in the different fields of the expression, like also `c`:
```sh
sed -i 'sc-fuse-ld=x86_64-apple-darwin14c-fuse-ld=/opt/bin/x86_64-apple-darwin14-ldcg' /opt/bin/x86_64-apple-darwin14-clang
```

Reply:
I didn't know that, fixed everywhere.